Assets/PatchKit Patcher/Scripts/AppData/Local/Pack1Meta.cs

```diff
@@ -46,6 +51,8 @@ public class FileEntry
         {
             public string Name { get; set; }
 
+            public string Hash { get; set; }
```

Comment:
Rename this to `NameHash`. The `Hash` itself is unclear what it relates to.

Reply:
Outdated, I changed this for something else